src/SocialFollow/__tests__/SocialFollow.test.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import expect from 'expect';
+import { shallow, mount } from 'enzyme';
+import SocialFollow from '../SocialFollow';
+
+const media = {
+  facebook: 'https://www.thebyte9.com',
+  twitter: 'https://www.thebyte9.com',
+  pinterest: 'https://www.thebyte9.com',
+  linkedIn: 'https://www.thebyte9.com',
+  youtube: 'https://www.thebyte9.com',
+  instagram: 'https://www.thebyte9.com'
+};
+
```

Comment:
If media is a constant (used both here and within the stories component), you could save it in the constant file and import it in each component instead of declaring media twice